bin/asstatd.pl

```diff
@@ -73,7 +73,15 @@
 	die("sFlow server port can't be the same as NetFlow server port\n");
 }
 
-die("Your own AS number is non numeric\n") if ($sflow_server_port > 0 && $myas !~ /^[0-9]+$/);
+my %myas;
+if($sflow_server_port > 0){
+	die('No ASN found, please specify -a') if !defined($myas_opt);
+	%myas = split(',', $myas_opt);
```

Comment:
Am I overlooking something, or doesn't this ignore every second AS specified in the comma-separated list (because it gets loaded into the hash as a value instead of as a key)?